eslint.config.mjs

```diff
@@ -0,0 +1,128 @@
+import eslintPluginVue from 'eslint-plugin-vue'
+import vuejsAccessibility from 'eslint-plugin-vuejs-accessibility'
+import eslintPluginUnicorn from 'eslint-plugin-unicorn'
+import eslintConfigPrettier from 'eslint-config-prettier'
+import intlifyVueI18N from '@intlify/eslint-plugin-vue-i18n'
+import globals from 'globals'
+import vueEslintParser from 'vue-eslint-parser'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import js from '@eslint/js'
+import { FlatCompat } from '@eslint/eslintrc'
+import { fixupConfigRules } from '@eslint/compat'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+})
+
+export default [
+  ...fixupConfigRules(
+    compat.config({
+      extends: ['standard']
+    })
+  ),
+  js.configs.recommended,
+  eslintConfigPrettier,
+  ...eslintPluginVue.configs['flat/vue2-recommended'],
+  ...vuejsAccessibility.configs["flat/recommended"],
+  ...intlifyVueI18N.configs['flat/recommended'],
+  {
+    files: [
+      '**/*.{js,vue}',
+    ],
+    ignores: [
+      '**/node_modules',
+      '**/_scripts',
+      '**/dist',
+    ],
+    plugins: {
+      unicorn: eslintPluginUnicorn,
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.node,
+      },
+
+      parser: vueEslintParser,
+      ecmaVersion: 2022,
+      sourceType: 'module',
+    },
+
+    settings: {
+      'vue-i18n': {
+        localeDir: './static/locales/{en-US,en-GB,ar,bg,ca,cs,da,de-DE,el,es,es-AR,es-MX,et,eu,fa,fi,fr-FR,gl,he,hu,hr,id,is,it,ja,ko,lt,nb-NO,nl,nn,pl,pt,pt-BR,pt-PT,ro,ru,sk,sl,sr,sv,tr,uk,vi,zh-CN,zh-TW}.yaml',
```

Comment:
Please import the list instead of hardcoding it, so that we don't have to maintain it in multiple places. Using `import activeLocales from './static/locales/activeLocales.json' with { type 'json' }` and then calling `.join(',')` on the imported array should work.

According to the history drop down on https://nodejs.org/api/esm.html#import-attributes the `with` keyword is supported in Node 21+ and in the LTS versions 18.20+ and 20.10+, so we should be fine using it.

Reply:
With dynamic import, got warning (probably non issue assuming it eventually no longer Experimental 
```js
const { default: activeLocales } =
  await import('./static/locales/activeLocales.json', { with: { type: 'json' } })
```
![image](https://github.com/user-attachments/assets/95efc2b6-51d7-46aa-823a-f5071ae1726c)

Also static import would only cause syntax error with node 20.17.0
![image](https://github.com/user-attachments/assets/ab6adda0-b5d7-4c66-a5df-29906c4c9872)
![image](https://github.com/user-attachments/assets/55f24e1d-eab3-42f4-9b1a-bca90f3de989)

Reply:
The reason it didn't work with the static import is because of the missing `:` between `type` and `'json'`.